test/projects_test.exs

```diff
@@ -135,4 +135,59 @@ defmodule GlassFactoryApi.ProjectsTest do
       end
     end
   end
+
+  describe "list_members/2" do
+    test "returns a tuple with ok and a list of ProjectMembers", %{bypass: bypass, config: config} do
+      request_response = GlassFactoryApi.Fixtures.ProjectMembers.list()
+
+      Bypass.expect_once(bypass, fn conn ->
```

Comment:
Now that our test suite is growing bigger we can think of extracting some test helpers for reuse.

Reply:
I'll open another PR for that!! 😄